pandas/tseries/common.py

```diff
@@ -106,6 +106,8 @@ def _delegate_property_get(self, name):
         elif not is_list_like(result):
             return result
 
+        result = np.asarray(result)
+
```

Comment:
the `take_1d` 2 lines below needs an array, not an index. 
I could only convert it specifically for that, but thought it couldn't do harm to put it here, as it is otherwise passed to Series as values, so will be converted to array anyway.